Svc/SeqDispatcher/docs/sdd.md

```diff
@@ -0,0 +1,66 @@
+# components::SeqDispatcher
```

Comment:
If this is to remain empty, then it should be deleted.  Otherwise fill in your design.